test/test_openai.py

```diff
@@ -1,3 +1,7 @@
+from . import supported
+
+
+@supported("python", version="1.14.0")
```

Comment:
@sabrenner in what version of the node library was the initial openai integration released?

Reply:
It was released in versions `4.4.0`, `3.25.0`, and `2.38.0`, not sure how that would fit into the version check, but `3.x` and `2.x` are no longer maintained, so probably safe to just use `4.4.0`

Reply:
added!